standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -364,32 +380,13 @@
       </field>
     </class>
 
-    <class name="MColumnDescriptor" identity-type="datastore" table="CDS" detachable="true">
-      <datastore-identity>
-        <column name="CD_ID"/>
-      </datastore-identity>
-      <field name="cols" persistence-modifier="persistent" table="COLUMNS_V2">
-        <collection element-type="MFieldSchema" dependent-element="true" />
-        <join>
-          <primary-key name="SQL110922153006740">
-            <column name="CD_ID"/>
-            <column name="COLUMN_NAME"/>
-          </primary-key>
-          <column name="CD_ID"/>
-        </join>
-        <element>
-          <embedded>
-            <field name="name">
-              <column name="COLUMN_NAME" length="767" jdbc-type="VARCHAR" allows-null="false"/>
-            </field>
-            <field name="type">
-              <column name="TYPE_NAME" length="32672" jdbc-type="VARCHAR" allows-null="false"/>
-            </field>
-            <field name="comment">
-              <column name="COMMENT" length="256" jdbc-type="VARCHAR" allows-null="true"/>
-            </field>
-          </embedded>
-        </element>
+    <class name="MColumnDescriptor" identity-type="application" table="CDS" detachable="true">
+      <field name="id" primary-key="true" value-strategy="native">
```

Comment:
The native strategy sounds good, though I'm not a specialist of JDO.
https://www.datanucleus.org/products/accessplatform_4_2/jdo/value_generation.html